src/frontend/src/lib/components/ui/Warning.svelte

```diff
@@ -1,10 +1,16 @@
 <script lang="ts">
-	import { IconWarning } from '@dfinity/gix-components';
+	import { IconInfo, IconWarning } from '@dfinity/gix-components';
+	import type { ComponentType } from 'svelte';
+
+	export let info = false;
+
+	let icon: ComponentType;
+	$: icon = info ? IconInfo : IconWarning;
```

Comment:
Feels weird to have a component named `Warning` that can be an `info`. What's the plan?

Reply:
I did that to avoid duplicating code. What would be a good name for that component? `AlertDialog` or `MessageDialog` or `AlertLabel`?

Reply:
We can have two separate components. `Warning.svelte` and `Info.svelte`.

Reply:
yes, maybe better eheh! separated